specification/communication/data-plane/Chat/preview/2024-03-15-preview/communicationserviceschat.json

```diff
@@ -0,0 +1,2068 @@
+      "x-ms-enum": {
+        "name": "ChatMessageType",
+        "modelAsString": true
+      },
+      "example": "html"
+    },
+    "SendChatMessageRequest": {
+      "description": "Details of the message to send.",
+      "required": [
+        "content"
+      ],
+      "type": "object",
+      "properties": {
+        "content": {
+          "description": "Chat message content.",
+          "type": "string",
+          "example": "<p>Come one guys, lets go for lunch together.</p>"
+        },
+        "senderDisplayName": {
+          "description": "The display name of the chat message sender. This property is used to populate sender name for push notifications.",
+          "type": "string",
+          "example": "Bob Admin"
+        },
+        "type": {
+          "$ref": "#/definitions/ChatMessageType"
+        },
+        "metadata": {
+          "description": "Message metadata.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "attachments": {
+          "description": "The array of attachments",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ChatAttachment"
+          }
+        }
+      }
+    },
+    "SendChatMessageResult": {
+      "description": "Result of the send message operation.",
+      "required": [
+        "id"
+      ],
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "A server-generated message id.",
+          "type": "string",
+          "example": "123456789"
+        }
+      }
+    },
+    "ChatParticipant": {
+      "description": "A participant of the chat thread.",
+      "required": [
+        "communicationIdentifier"
+      ],
+      "type": "object",
+      "properties": {
+        "communicationIdentifier": {
+          "$ref": "#/definitions/CommunicationIdentifierModel"
+        },
+        "displayName": {
+          "description": "Display name for the chat participant.",
+          "type": "string",
+          "example": "Bob"
+        },
+        "shareHistoryTime": {
+          "format": "date-time",
+          "description": "Time from which the chat history is shared with the participant. The timestamp is in RFC3339 format: `yyyy-MM-ddTHH:mm:ssZ`.",
+          "type": "string",
+          "example": "2020-10-30T10:50:50Z"
+        },
+        "metadata": {
+          "description": "Contextual metadata for the chat participant. The metadata consists of name/value pairs. The total size of all metadata pairs can be up to 1KB in size.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "ChatAttachmentType": {
+      "description": "The type of attachment.",
+      "enum": [
+        "image",
+        "file"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "ChatAttachmentType",
+        "modelAsString": true
+      }
```

Comment:
Include descriptions for all values in the x-ms-enum extension.

Reply:
reverting this change because CI is complaining
<img width="1167" alt="image" src="https://github.com/Azure/azure-rest-api-specs/assets/77021369/b133e5f8-290a-47f0-bfe8-e2858643195b">